.github/CODEOWNERS

```diff
@@ -1,3 +1,11 @@
-* @artemrys
-/splunk_add_on_ucc_framework/schema/schema.json @artemrys @vtsvetkov-splunk
-/ui/ @vtsvetkov-splunk
+*                                                   @splunk/ucc-developers
+
+/ui/                                                @splunk/ucc-ui-developers
+*.ts                                                @splunk/ucc-ui-developers
+*.tsx                                               @splunk/ucc-ui-developers
+
+*.py                                                @splunk/ucc-be-developers
+/tests                                              @splunk/ucc-be-developers
+/splunk_add_on_ucc_framework                        @splunk/ucc-be-developers
+
+/splunk_add_on_ucc_framework/schema/schema.json     @splunk/ucc-developers
```

Comment:
We probably do we need this as there is 
```*                                                   @splunk/ucc-developers```
but maybe worth leaving it just for visibility

Reply:
yea, what i meant is that, this * (everything is the very first line in changes), 

and i made a typo there it should be sth like. We probably do NOT need this line wich schema.json, as we got this first line with 
```*    @splunk/ucc-developers```
so that last line probably is not making any changes there but is only informative kind of